src/Configuration/RectorConfigBuilder.php

```diff
@@ -154,6 +155,10 @@ final class RectorConfigBuilder
      */
     private array $groupLoadedSets = [];
 
+    private bool $isWithPhpSetsUsed = false;
+
+    private bool $isWithPhpLevelUsed = false;
```

Comment:
same with `parallel`, `isCodeQualityLevelUsed` and other, this needs to be nullable bool, so it never reset on multiple config merge:

```suggestion
    private ?bool $isWithPhpSetsUsed = null;

    private ?bool $isWithPhpLevelUsed = null;
```

Reply:
Got it :+1: